app/services/processors/blog_technology_views_full_updater.rb

```diff
@@ -0,0 +1,9 @@
+module Processors
+  class BlogTechnologyViewsFullUpdater < BlogTechnologyViewsUpdater
+    private
+
+    def metrics_by_timestamp
+      Metric
```

Comment:
Should this return `Metric.all`, so that it is consistent with what the other `metrics_by_timestamp` method returns (an Arel)

Reply:
Sure!